scripts/tests/test_validate_docstrings.py

```diff
@@ -407,6 +407,20 @@ def sections_in_wrong_order(self):
             before Examples.
         """
 
+    def deprecation_in_wrong_order(self):
+        """
+        This is what old method does.
+
+        This is an extended summary with more details about
+        what `some_old_method` does.
+
+        The extended summary can contain multiple paragraphs.
```

Comment:
Do you mind of instead of this description, explain what is wrong in the PR here.

Reply:
Have changed the description to explain what is wrong with the Docstring